ecocode-rules-specifications/pom.xml

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>io.ecocode</groupId>
+		<artifactId>ecocode-parent</artifactId>
+		<version>1.2.2-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>ecocode-rules-specifications</artifactId>
+
+	<name>ecoCode Rules Specifications repository</name>
+	<description>Repository that contains the specifications of every static-analysis rules available in ecoCode plugins.</description>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.sonarsource.sonarqube</groupId>
+			<artifactId>sonar-plugin-api</artifactId>
+			<scope>provided</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.sonarsource.analyzer-commons</groupId>
+			<artifactId>sonar-analyzer-commons</artifactId>
+			<scope>provided</scope>
+		</dependency>
+	</dependencies>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.asciidoctor</groupId>
+				<artifactId>asciidoctor-maven-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>convert-to-html</id>
+						<phase>generate-resources</phase>
+						<goals>
+							<goal>process-asciidoc</goal>
+						</goals>
+						<configuration>
+							<sourceDirectory>${project.basedir}/src/main/rules</sourceDirectory>
+							<outputDirectory>${project.build.directory}/rules</outputDirectory>
+							<attributes>
+								<source-highlighter>coderay</source-highlighter>
+								<coderay-css>style</coderay-css>
+							</attributes>
+							<preserveDirectories>true</preserveDirectories>
+							<headerFooter>false</headerFooter>
+							<enableVerbose>true</enableVerbose>
+							<relativeBaseDir>true</relativeBaseDir>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-antrun-plugin</artifactId>
+				<version>3.1.0</version>
+				<executions>
+					<execution>
+						<phase>process-resources</phase>
```

Comment:
How this will work when we will have multiples repositories (one per language)?
I think we should already think about the new structure because we will move to it in a near future

Reply:
I think we can use solution described in: https://stackoverflow.com/questions/64878131/use-resourceslike-jar-files-of-a-github-repo-in-another-repo-using-github-acti

Reply:
We will publish this dependency on Maven Central to facilitate sharing